miguel/block-one.js

```diff
@@ -0,0 +1,94 @@
+// In this file we are going to write the problems. Let's keep all in english to practice that as well.
+
+// 1. Create a function that returns me an array of 10 names with alphabetically order.
+
+// 2. Given an array of numbers, I want the sum of all of those numbers:
+    // Example: [10, 6, 8] be equal 24.
+
+// 3. You have an array of person objects, where each object contains a name and age.
+// Your task is to calculate the average age of all the people in the array and return the result.
+    // Example: const people = [
+//   { name: 'John', age: 25 },
+//   { name: 'Jane', age: 30 },
+//   { name: 'Sam', age: 20 },
+//   { name: 'Alice', age: 35 },
+//   { name: 'Bob', age: 27 }
+// ];
+
+
+//This is the original array of names
+const names = ['Ralph', 'Joe', 'Mark','Sophie', 'Alvin', 'Laura', 'Kathy','Dean','Lisa', 'Tom']
+
+//This is the first solution I mistakenly used to solve this problem
+//a simple .sort() stored withtin another constant
+let orderedNames = names.sort();
+
+console.log(orderedNames);
+
+//This is the correct solution using a function 
+//To be completely fair, I looked up this solution and I'm still trying to 
+//understand the logic used as to how the order is changed
+
+//On the bright side : I know how to google, hence I'm ready to work (jk)
+const newOrder = names.sort(function (a , b) {
```

Comment:
Remember that a.names is no needed since there is no names property in the array, just a < b